orttraining/orttraining/test/python/orttraining_test_ortmodule_api.py

```diff
@@ -16,8 +16,9 @@
 from collections import namedtuple
 from inspect import signature
 import tempfile
+import os
 
-from onnxruntime.training.ortmodule import ORTModule, _utils, _io
+from onnxruntime.training.ortmodule import ORTModule, _utils, _io, DebugOptions, LogLevel
```

Comment:
make sure both env vars and explicit setting are tested for success and error and append something to the name to distinguish them